ddp-workspace/projects/ddp-testboston/src/app/components/help/help.component.html

```diff
@@ -6,7 +6,7 @@ <h1 class="center" translate>Help.Title</h1>
 
                 <h2 class="sub-title" translate>Help.Collect.Title</h2>
                 <p translate>Help.Collect.Text</p>
-                <a class="pdf" href="kit-instructions.pdf" target="_blank">
+                <a class="pdf" href="en/kit-instructions.pdf" target="_blank">
```

Comment:
I'm gonna need help adding in the language switching here.

Reply:
Done